### PR TITLE
Fix windows pypi pipeline and add win32 version

### PR DESCRIPTION
**Pull request by demianzhang**
1. Add Windows 32-bit version for source install and pip install.
2. Change the pypi pipeline script to powershell

Tested in virtual machine Windows 10 32-bit using virtualbox.

### CODE REVIEW

File: deployment/pypi/README.md [hunk 1]
```diff
@@ -53,9 +53,10 @@ This is the PyPI build and upload tool for NNI project.
   ```
 
 * __How to build__
-
+ 
+  parameter `version_os` is used to choose Windows version 64-bit or 32-bit, `version_ts` is used to build for test or not. 
   ```bash
-  powershell ./install.ps1
+  powershell ./install.ps1 -version_os [64/32] -version_ts [$True/$False]
```
Comment (chicm-ms) [inline deployment/pypi/README.md]: Can you remove the -version_ts option from doc?   it could be change or removed from code.

Comment (demianzhang) [reply deployment/pypi/README.md]: I can not understand the change. Could you give more details?

Comment (chicm-ms) [reply deployment/pypi/README.md]: -version_ts option is an implementation detail,  should not be mentioned in document.

Comment (demianzhang) [reply deployment/pypi/README.md]: It decides whether to build pypi package for test or release. Should we remove the parameters -version_os and -version_ts?

**Pull request merged**